src/app/app.module.ts

```diff
@@ -25,6 +26,7 @@ import { DashboardModule } from './dashboard/dashboard.module';
     routing,
     FormsModule,
     AlertsModule,
+    AlertModule.forRoot(),
```

Comment:
I'll rename all imports of the third-party `AlertModule` with an alias to `BootstrapAlertModule`. Not sure this is the best solution but at least it keeps things clear and distinguishable.